packages/@aws-cdk/aws-rds/test/cluster.test.ts

```diff
@@ -721,6 +721,66 @@ describe('cluster', () => {
 
   });
 
+  test('add single user rotation for master secret', () => {
+    // GIVEN
+    const stack = new cdk.Stack();
+    const vpc = new ec2.Vpc(stack, 'VPC');
+    const cluster = new DatabaseCluster(stack, 'Database', {
+      engine: DatabaseClusterEngine.AURORA_MYSQL,
+      credentials: { username: 'admin' },
+      instanceProps: {
+        instanceType: ec2.InstanceType.of(ec2.InstanceClass.BURSTABLE2, ec2.InstanceSize.SMALL),
+        vpc,
+      },
+    });
+
+    // WHEN
+    cluster.addRotationSingleUser();
```

Comment:
I feel like this case is already covered well enough in unit and integration tests, that I would remove this test.

Reply:
I can't find any unit test for it. But sure, if you say so.